libres/lib/enkf/local_obsdata_node.cpp

```diff
@@ -79,12 +79,28 @@ void local_obsdata_node_free__(void *arg) {
     local_obsdata_node_free(node);
 }
 
-active_list_type *
+ActiveList *
 local_obsdata_node_get_active_list(const local_obsdata_node_type *node) {
     return node->active_list;
 }
 
-active_list_type *
+ActiveList *
 local_obsdata_node_get_copy_active_list(const local_obsdata_node_type *node) {
-    return active_list_alloc_copy(node->active_list);
+    return new ActiveList(*node->active_list);
+}
+
+namespace {
+ActiveList &get_active_list(py::handle obj) {
+    auto *obsdata_node = reinterpret_cast<local_obsdata_node_type *>(
```

Comment:
Could be replaced with: `ert::from_cwrap<local_obsdata_node_type>(obj);`

Reply:
OK